glossary.md

```diff
@@ -0,0 +1,14 @@
+# Glossary
+
+### APY
+
+Annual Percentage Yield, is the normalized representation of an interest rate,
+based on a compounding period over one year. Note that APYs provided are rather
+ballpark level indicators and not so much precise future results.
+
+### BCV
+
+Bond Control Variable, is the scaling factor at which bond prices change. A
+higher BCV means a lower discount for bonders and less inflation by the
+protocol. A lower BCV means a higher capacity for bonders and less profit for
+the protocol.
```

Comment:
I might miss something here. My understanding is: Higher BCV --> lower discount --> attract less bonders --> less profit for protocol --> lower OHM inflation. Lower BCV --> higher discount --> attract more bonders --> more profit for the protocol --> higher OHM inflation.

But the definition states that a lower BCV translates to less profit for the protocol. Is it because we sell the OHM at too great a discount that the protocol can only make a small profit despite a large demand of bond?

Reply:
This is not clear, I agree. I actually just copied [this definition from the docs we already have](https://docs.olympusdao.finance/policy#bonds). There might be a factor of resolution here. As if you compare a single bond purchase relative to all bond purchases of a cycle. I would like to keep the glossary simple so we may want to drop this added complexity. But I am open to suggestions on how to best phrase this.

Reply:
@Wartull1337 you got an opinion on this? Feel free to suggest your own line of thought.

Reply:
At a low BCV, we get less inflow (because we sell at a higher discount) which in turn generates less OHM which can be minted so lower inflation

Reply:
Alright I think I rewrote it in the correct way. Please have another look.